supervisor/utils/common.py

```diff
@@ -16,12 +16,14 @@
 
 
 def find_one_filetype(
-    path: Path, filename: str, filetypes: List[str]
+    path: Path, filename: str, filetypes: List[str], default: Optional[str] = None
```

Comment:
let's call it `as_new=None`, so we can call the function with `self.addon.path_location, "build", FILE_SUFFIX_CONFIGURATION, as_new="json"` and in next step we can raise an exception instead return None if `as_new` is not set, which make the error handling simpler

Reply:
nvm, I make an new PR :)

Reply:
feel not good to make this function so many use case